### PR TITLE
pypy url_base naming convention to match flannel and kube

### PR DESCRIPTION
**Pull request by stephenrlouie**
- Finally clean PR

@rutsky @danehans @eparis. Could you all check this out? 

Rebase on #723

### CODE REVIEW

File: ansible/roles/pre-ansible/defaults/main.yml [hunk 1]
```diff
@@ -6,4 +6,6 @@ bin_dir: /usr/bin
 # reside.
 bootstrap_script_dir: /opt/bin
 pypy_version: 2.4.0
-pypy_base_url: https://bitbucket.org/pypy/pypy/downloads
+
+pypy_download_url_base: "https://bitbucket.org/pypy/pypy/downloads"
+pypy_download_url: "{ pypy_download_url_base }}/pypy-{ pypy_version }}-linux64.tar.bz2"}}
```
Comment (rutsky) [inline ansible/roles/pre-ansible/defaults/main.yml]: @stephenrlouie I see that opening and closing brackets are messed here.

Have you tried these changes in clear environment?

**Comment by danehans** [comment]
@stephenrlouie IMO this merged PR addresses the issue: https://github.com/kubernetes/contrib/pull/744

File: ansible/roles/pre-ansible/defaults/main.yml (hunk 1 above)
Comment (stephenrlouie) [inline ansible/roles/pre-ansible/defaults/main.yml]: @rutsky Good catch, I did not test this in a clear environment, give me a few and I'll run it in vagrant. 

Also caught a mistake on the end of the url with the }}

File: ansible/roles/pre-ansible/tasks/coreos.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@
          http_proxy="{{ http_proxy|default('') }}"
          https_proxy="{{ https_proxy|default('') }}"
          no_proxy="{{ no_proxy|default('') }}"
-         wget -O /tmp/pypy-{{ pypy_version }}-linux64.tar.bz2 {{ pypy_base_url }}/pypy-{{ pypy_version }}-linux64.tar.bz2
+         wget -O /tmp/pypy-{{ pypy_version }}-linux64.tar.bz2 {{ pypy_download_url }}
```
Comment (rutsky) [inline ansible/roles/pre-ansible/tasks/coreos.yml]: I think there is unnecessary trailing space.

**Comment by danehans** [comment]
@stephenrlouie I always do a git diff --check to check trailing whitespace before git add/commit.

**Comment by stephenrlouie** [comment]
Works now

**Comment by stephenrlouie** [comment]
@danehans. It is similar, the distinction for this one is to bring naming consistency across the url download variables.

Without this PR we have
```
kube_download_url_base
flannel_download_url_base
pypy_base_url
```

With the PR they would match, this PR also brings all the information into one file /defaults/main.yml.

**Comment by danehans** [comment]
LGTM

/cc: @eparis 

@stephenrlouie please update the gemini repo accordingly. Thanks!

**Pull request merged**